entities/package.py

```diff
@@ -0,0 +1,102 @@
+from typing import List
+from common import format_float
+from .common import Name, Description, Keywords, Author, Version, Created, Category, Deprecated, Position, Rotation, Polygon, Layer, Height, Align, Value, FloatValue, EnumValue, BoolValue
+
+
+class Pad():
+    def __init__(self, uuid: str, name: Name):
+        self.uuid = uuid
+        self.name = name
+
+
+class StrokeWidth(FloatValue):
+    def __init__(self, stroke_width: float):
+        super().__init__('stroke_width', stroke_width)
+
+
+class LetterSpacing(EnumValue):
+    AUTO = 'auto'
+
+    def get_name(self) -> str:
+        return 'letter_spacing'
+
+
+class LineSpacing(EnumValue):
+    AUTO = 'auto'
+
+    def get_name(self) -> str:
+        return 'letter_spacing'
+
+
+class AutoRotate(BoolValue):
+    def __init__(self, auto_rotate: bool):
+        super().__init__('auto_rotate', auto_rotate)
+
+
+class Mirror(BoolValue):
+    def __init__(self, mirror: bool):
+        super().__init__('mirror', mirror)
+
+
+class StrokeText():
+    def __init__(self, uuid: str, layer: Layer, height: Height, stroke_width: StrokeWidth, letter_spacing: LetterSpacing, line_spacing: LineSpacing, align: Align, position: Position, rotation: Rotation, auto_rotate: AutoRotate, mirror: Mirror, value: Value):
+        pass
+
+
+class Side(EnumValue):
+    THT = 'tht'
+
+    def get_name(self) -> str:
+        return 'side'
+
+
+class Shape(EnumValue):
+    ROUND = 'round'
+    RECT = 'rect'
```

Comment:
Is this the pad shape? I think there are octagon shapes too.

Reply:
We can add it when we use it. I currently only add stuff which is actually used in the generated output, so it is tested.

Reply:
I added the other values anyways.